lab-andrew/test/reminder-route-test.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
```

Comment:
This is a good start, your code could be refactored significantly and there are a few tests missing.  Instead of creating a reminder each time you perform a route test, I suggest you create one reminder in your POST request, then use it for all your other tests.  Organizing them like this:

GENERAL - 404
POST
GET
PUT
DELETE

This way you will only create one reminder which will be deleted at the end of your testing, giving each test run a clean start.  Besides that, you're missing tests for your storage module and delete routes.